.github/workflows/labeling-governance.yml

```
@@ -230,3 +230,119 @@ jobs:
+    name: Labeling Governance Check (Required)
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    needs: [standard-labeling]
+    permissions:
+      checks: write
+      pull-requests: write
+
+    steps:
+      - name: Check labeling workflow status
+        id: check
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:28: Double quote to prevent globbing and word splitting

(shellcheck)